pages/support.js

```diff
@@ -9,48 +9,77 @@ import resourcesData from '../SupportResources.js';
 
 export default function Support() {
   const [resourcesList, setResourcesList] = useState(resourcesData);
+  const [allyship, setAllyship] = useState("anyone")
+  const [category, setCategory] = useState("All")
 
-  const filterResourcesByAllyship = (allyship) =>{
-
-    if(allyship === 'anyone'){
-      setResourcesList(resourcesData);
-      return;
-    }
+  const changeAllyship = (chosenAllyship) => {
+    // console.log("chosenAllyship: ", chosenAllyship)
+    setAllyship(chosenAllyship)
+  }
 
-    const filteredResources = resourcesData.filter(resource => resource.category[0] === buttonCategory || resource.category[1] === buttonCategory);
-    setResourcesList(filteredResources)
+  const changeCategory = (chosenCategory) => {
+    // console.log("chosenCategory: ", chosenCategory)
+    setCategory(chosenCategory)
   }
 
-  const filterResourcesByCategory = (buttonCategory) =>{
+  const filterResources = () =>{
+    if (allyship === "anyone" && category === "All") {
+      return resourcesData
+    } else if (allyship === "anyone" && category !== "All") {
+      let filteredResourcesByCategory = resourcesData.filter(resource => {
+        return resource.category[0] === category || resource.category[1] === category
+      })
 
-    if(buttonCategory === 'All'){
-      setResourcesList(resourcesData);
-      return;
```

Comment:
Fantastic changes, Wavey! In Support.js, we may be able to use a switch case starting at line 25, but if not, no problem. Thank you for the amazing work!

Reply:
Thanks Manny! I'm not quite sure how to handle the logic for cases, specifically how to keep track and change the case based on when the user clicks a different allyship/category. We can keep this in mind for the future, or if you have a solution you can tackle it!